.github/workflows/syrius_builder.yml

```diff
@@ -28,8 +28,8 @@ jobs:
       - name: Setup Flutter
         uses: subosito/flutter-action@v2.10.0
         with:
+          flutter-version: ${{env.FLUTTER_VERSION}}
```

Comment:
Is this variable set in the zenon-network repo?

Reply:
There are no variables for this repository’s environments.

Reply:
I missed the variable declaration the first time. No need to set anything.